clients/drcachesim/tracer/raw2trace.h

```diff
@@ -1304,6 +1329,20 @@ template <typename T> class trace_converter_t {
         }
         impl()->log(4, "Appended memref type %d size %d to " PFX "\n", buf->type,
                     buf->size, (ptr_uint_t)buf->addr);
+
+#ifdef AARCH64
+        // TODO i#4400: Following is a workaround to correctly represent DC ZVA in
+        // offline traces. This doesn't help with online traces, and also causes us
+        // to lose the actually written address. Ideally, we want the actually
+        // written address and also the correct size and type in the IR.
```

Comment:
"actually written address" seems a misnomer since it's not what is (only) written, and the trace doesn't need it -- unless you had some reason in mind to need it?  The only place I can think of where we might want the originally specified address in the instruction is in the encoder for re-encoding fidelity which for most DR operations is considered a corner case (i#3339, as referenced in i#4400).

Reply:
Updated the comment.